giftless/storage/local_storage.py

```diff
@@ -42,6 +43,13 @@ def get_size(self, prefix: str, oid: str) -> int:
             return os.path.getsize(self._get_path(prefix, oid))
         raise exc.ObjectNotFound("Object was not found")
 
+    def get_mime_type(self, prefix: str, oid: str) -> str:
```

Comment:
I think this will not work, because `mimetypes` uses the file name to guess the type, and this will not have any relevant file name (just sha256 no extension). Maybe it's best to fall back to `application/octet-stream` here.